tern/formats/spdx_new/make_spdx_model.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2021 VMware, Inc. All Rights Reserved.
```

Comment:
small nit: copyright year needs updating

Reply:
updated to 2023